src/db.rs

```diff
@@ -97,17 +110,27 @@ where
         let store_config = Config::default()
             .with_size_targets(size)
             .with_pragma_synchronous(Synchronous::Normal);
+        let mk_cache_entry = |access, _| Some(access);
+        let tracker = if let Some(path) = config.access_db_path {
+            let path = if path.is_file() {
+                path
+            } else {
+                std::fs::create_dir_all(&path)?;
+                path.join("access")
```

Comment:
this is called access, not db, so you can use the same directory for both dbs if you want to.

Reply:
I think the pattern here results in unexpected behaviour. If the file exists, use it, if not create a folder and create a file in it..

Reply:
I never understood the rationale for interpreting the path differently based on whether it exists and/or is a file, but I guess that ship has sailed. I’d greatly simplify the semantics of these parameters by always interpreting them as the precise file path where the db lives.